packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -73,12 +73,22 @@ class FirebaseAuth {
     'plugins.flutter.io/firebase_auth',
   );
 
+  StreamController<Null> _onAuthStateChanged;
```

Comment:
I would `StreamController<FirebaseUser>` here and call it `_onCurrentUserChanged`